concent-builder/deploy.yml

```diff
@@ -0,0 +1,36 @@
+- hosts:
+    - concent-builder
+  vars_files:
+    - consts.yml
+  tasks:
+    - become:      yes
+      remote_user: builder
+      block:
+        - name:    Google Cloud Engine service account activating
+          command: gcloud auth activate-service-account concent-builder-service@devel-concent-golem-network.iam.gserviceaccount.com --key-file "/home/{{ shared_user }}/build/concent-secrets/devel-concent-golem-network-198b161e7c8a.json"
+
+        - name:    Google Cloud Engine service prepare project
+          command: gcloud container clusters get-credentials concent-dev --zone europe-west3-a --project devel-concent-golem-network
```

Comment:
Name of the project, cluster and zone should all come from variables defined in `var.yml` in `concent-deployment-values`.